agimus_controller/robot_model/ur5_model.py

```diff
@@ -0,0 +1,36 @@
+from math import pi
+import numpy as np
+from pathlib import Path
+import rospkg
+from agimus_controller.robot_model.robot_model import RobotModelParameters
+from agimus_controller.robot_model.robot_model import RobotModel
+
+
+class UR5RobotModelParameters(RobotModelParameters):
```

Comment:
the hpp script we use is for ur3, so we should be loading the model for it here

Reply:
Fixed